gnovm/stdlibs/stdlibs.go

```diff
@@ -13,11 +13,31 @@ func GetContext(m *gno.Machine) ExecContext {
 	return libsstd.GetContext(m)
 }
 
-func NativeStore(pkgPath string, name gno.Name) func(*gno.Machine) {
+func findNative(pkgPath string, name gno.Name) nativeFunc {
 	for _, nf := range nativeFuncs {
 		if nf.gnoPkg == pkgPath && name == nf.gnoFunc {
-			return nf.f
+			return nf
 		}
 	}
-	return nil
+	return nativeFunc{}
+}
+
+// NativeStore is used by the GnoVM to determine if the given function,
+// specified by its pkgPath and name, has a native implementation; and if so
+// retrieve it.
+func NativeStore(pkgPath string, name gno.Name) func(*gno.Machine) {
+	return findNative(pkgPath, name).f
+}
+
+// HasNativeBinding determines if the function specified by the given pkgPath
+// and name is a native binding.
+func HasNativeBinding(pkgPath string, name gno.Name) bool {
+	return findNative(pkgPath, name).f != nil
+}
```

Comment:
we can use that boolean on these kinds of functions. It will represent the not-found behavior much better IMHO. You can even delete this function and use `findNative` instead